<?xml version="1.0" encoding="UTF-8"?>  
<launch>
    <arg name="goal_x" default="0"/>
    <arg name="goal_y" default="12"/>
    <arg name="goal_z" default="0.25"/>
    <arg name="move_precision" default="0.1"/>
    <arg name="velocity" default="0.056"/>
    
    <node pkg="marker_move" type="go_to_goal" name="go_to_goal" output="screen">
    	<param name="Goal_x" value="$(arg goal_x)"/>
    	<param name="Goal_y" value="$(arg goal_y)"/>
    	<param name="Goal_z" value="$(arg goal_z)"/>
    	<param name="Move_precision" value="$(arg move_precision)"/>
    	<param name="Velocity" value="$(arg velocity)"/>
    </node>

  </launch> 
